project/Dependencies.scala

```diff
@@ -29,6 +29,9 @@ object Dependencies {
     "xml-resolver" % "xml-resolver" % "1.2",
     "commons-io" % "commons-io" % "2.6",
     "jline" % "jline" % "2.14.6",
+  )
+
+  lazy val junit = Seq(
     "junit" % "junit" % "4.12", // needed for TDML runner use of AssertionViolationException
```

Comment:
Minor, but junit is now in the file twice, once here and one in ``lazy val test``. The concern is that if we update the junit version we might not realize it needs to be updated in two places. An alternative is to not define junit as a Seq, but define it as a single dependency, then use that both here and in build.sbt, for example:
```scala
// Dependencies.scala
lazy val junit = "junit" % "junit" % "4.12"

lazy val test = Seq(
  junit % "it,test",
  "com.novocode" % "junit-interface" % "0.11" % "it,test",
  "org.scalacheck" %% "scalacheck" % "1.14.0" % "it,test"
)

// build.sbt
lazy val tdml = ...
                .settings(libraryDependencies += junit)  // note += instead of ++=
```
This way a version update is just in a single place.

Reply:
Fixed.